.github/pull_request_template.md

```diff
@@ -6,3 +6,12 @@
 ## Solution
 
 - Describe the solution used to achieve the objective above.
+
+## Changelog
```

Comment:
```suggestion
## \[ Optional \] Changelog
```

IMO this should be optional: trivial bug fixes, doc improvements and so on don't need anything beyond Problem and Solution.

Reply:
If we take this route, we should explain *why* it's optional in the template.